src/pages/Admin/Sidebar/Header.tsx

```diff
@@ -2,9 +2,11 @@ import { useProfileQuery } from "services/aws/aws";
 import ContentLoader from "components/ContentLoader";
 import Icon from "components/Icon";
 import Logo from "components/Logo";
+import { useAdminResources } from "../Guard";
 
-export default function Header({ endowId }: { endowId: number }) {
-  const { data: profile, isLoading, isError } = useProfileQuery(endowId);
+export default function Header() {
+  const { id } = useAdminResources();
+  const { data: profile, isLoading, isError } = useProfileQuery(id);
```

Comment:
@SovereignAndrey do we hide sidebar header when the connected wallet belongs to either AP or Review team?

Reply:
@misicnenad @ap-justin Sidebar header would NOT be hidden for AP Team nor for the Review Team. In reality, we'll probably want to have a set header that would act as an extra cue for those users as to which Admin area they are in and avoid accidental proposals to the wrong contracts. An example cue might be a solid header background color that is different for each ap admin type.

Reply:
@SovereignAndrey what would we display in Header for AP and Review teams?
- Logo: can be displayed -> could add some additional text to it as extra cue, like "Endowment", "AP Team", "Review".
- Endowment name -> makes little sense for them, do we display something else here?
- Endowment address -> what would we display here? 

All in all, we could have a header with different background colors and maybe difference logos depending on what type of wallet is connected.

Reply:
@misicnenad 

> @misicnenad @ap-justin Sidebar header would NOT be hidden for AP Team nor for the Review Team.

in the mininum, kindly apply these changes 
> so useProfileQuery should be skipped if adminResource.type !== "charity"

since `AP` and `REVIEW` ids doesn't pertain to any endowment-id

Reply:
@ap-justin will update, just waiting on Andrey's comments on the header subcomponents described above.

Reply:
> @SovereignAndrey what would we display in Header for AP and Review teams?
> logo 
Use a simple placeholder like angelwings image?

> - Endowment name -> makes little sense for them, do we display something else here?
Nothing here, unless here's where we would put the above mentioned extra text: `"Angel Giving Team Admin", "Charity Applications Review".`

> - Endowment address -> what would we display here? 
Nothing needs to go here for now. 

> 
> All in all, we could have a header with different background colors and maybe difference logos depending on what type of wallet is connected. 
👍

Reply:
@SovereignAndrey different color examples (subject to change, these were just the first colors that popped to mind, not much thought went into them):
- Charities - as-is:
![image](https://user-images.githubusercontent.com/19427053/220546318-512e2db5-027f-49aa-8715-ea117da27866.png)
![image](https://user-images.githubusercontent.com/19427053/220546288-f5aa11b1-364f-4dd6-a381-2e00fff5b5c4.png)
- AP team:
![image](https://user-images.githubusercontent.com/19427053/220545931-52ceded6-04c0-41dd-8119-5c6add579642.png)
![image](https://user-images.githubusercontent.com/19427053/220545958-5da20cbe-617a-489f-9d0d-41d95cfb95a5.png)
- Review team:
![image](https://user-images.githubusercontent.com/19427053/220545826-3dbff471-2884-4879-8347-5b9cac341279.png)
![image](https://user-images.githubusercontent.com/19427053/220545850-40449e64-a641-4832-ae70-bbf508249b3d.png)

Reply:
> * AP team:
>   ![image](https://user-images.githubusercontent.com/19427053/220545931-52ceded6-04c0-41dd-8119-5c6add579642.png)

The yellow is a bit jarring. Let's make both admins' colors consistent, regardless of light/dark mode. I would use the below dark mode colors for both light & dark receptively. 

> * AP team:
>   ![image](https://user-images.githubusercontent.com/19427053/220545958-5da20cbe-617a-489f-9d0d-41d95cfb95a5.png)
> * Review team:
>   ![image](https://user-images.githubusercontent.com/19427053/220545850-40449e64-a641-4832-ae70-bbf508249b3d.png)

Reply:
Updated so that all headers have same background color on light mode, but dark colors are as in last @SovereignAndrey's comment .